### DISCUSSION TITLE
Update of github action from 1.0.2 to 1.1.0 breaks build

### DISCUSSION THREAD

**Discussion by CarrotManMatt** [blue-build/.github#12] [Support] [answered]
Dependabot has alerted me that there is an update to the github action from 1.0.2 to 1.1.0. The pull request that makes this update automatically ran the build process but this build failed due to: "invalid reference format".

The failing build: https://github.com/CarrotManMatt/carrot-kinoite-nvidia/actions/runs/8055478674/job/22002432608
The attached update pull request: CarrotManMatt/carrot-kinoite-nvidia#2

**Comment by gmpinder** [comment, marked as answer]
This is actually a bug in the CLI. We aren't making sure to replace `/` in the generated tags

```
ghcr.io/carrotmanmatt/carrot-kinoite-nvidia:br-dependabot/github_actions/blue-build/github-action-1.1.0-39
```

Could you open an issue there so we can track it?

**Reply by CarrotManMatt** [reply]
Issue now tracked in blue-build/cli#92 😊
